vmm/src/memory_manager.rs

```diff
@@ -1244,6 +1244,20 @@ impl MemoryManager {
         0
     }
 
+    pub fn get_zone_actual(&self, id: &str) -> Result<u64, Error> {
```

Comment:
I use "actual" in this part because this part really like virtio-balloon.

After CLH set new size to "config.requested_size" and send signal to virtio-mem device in guest kernel, virtio-mem device will try to find appropriate pages and send them address to CLH with VIRTIO_MEM_REQ_PLUG and VIRTIO_MEM_REQ_UNPLUG.
Then CLH will handle this request and do the memory operation if need and do size record work.  The recorded size is the actual size.

And there is another hotplugged_size in MemoryConfig which will be updated after resize operation.

Reply:
Updated.